assets/i18n/en.json

```diff
@@ -519,16 +519,41 @@
     "I18N_SIGNUP_WHY_LICENSE": "Why CC-BY-SA?",
     "I18N_SOLICIT_ANSWER_DETAILS_FEEDBACK": "Okay, now let's go back to your answer.",
     "I18N_SOLICIT_ANSWER_DETAILS_QUESTION": "Could you explain why you picked this answer?",
+    "I18N_SPLASH_BENEFITS_ONE": "Personalized Learning",
+    "I18N_SPLASH_BENEFITS_THREE": "Easy-to-follow Lessons",
+    "I18N_SPLASH_BENEFITS_TITLE": "Our Benefits",
+    "I18N_SPLASH_BENEFITS_TWO": "Story-Based Lessons",
     "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "Oppia's lessons, also known as explorations, provide more immersive experiences than static videos or text, helping users learn by doing.",
+    "I18N_SPLASH_FOR_STUDENTS": "For Students",
+    "I18N_SPLASH_FOR_TEACHERS": "For Teachers",
+    "I18N_SPLASH_FOR_VOLUNTEERS": "For Volunteers",
+    "I18N_SPLASH_ICON_ONE_TEXT": "1 Million+ Users",
+    "I18N_SPLASH_ICON_THREE_TEXT": "40 Curated Lessons",
```

Comment:
This number is going to change ... should we put 40 in a variable instead and have it be interpolated accordingly in the translation, similar to `<[title]>` in I18N_STORY_VIEWER_COMPLETED_CHAPTER? Ditto for languages in the next field.

This will avoid the translations having to be retranslated when the numbers change. If you do this, please also add a note in qqq.json that the `<[...]>` part represents a number and should be left as-is when translating, though its position in the translated text may change.

Reply:
Done